README.md

```diff
@@ -41,6 +41,19 @@ The general overview of what you will need to do:
    Or, you can use a service such as https://ngrok.com/ to access on your local dev machine via localhost.
    Installation is fairly simple, though requires setting up a (free) account.
    Run the command `ngrok http 8000` to forward to port 8000 on localhost.
+
+   > Note: GitHub has a webhook forwarding service available in beta.
+   > See [cli/gh-webhook](https://docs.github.com/en/developers/webhooks-and-events/webhooks/receiving-webhooks-with-the-github-cli) for more information.
+   > This is super easy to use, and doesn't require manually configuring webhook settings.
+   > The command to run looks something like:
+   >
+   > ```sh
+   > gh webhook forward --repo=ehuss/triagebot-test --events=* \
+   >     --url=http://127.0.0.1:8000/github-hook --secret somelongsekrit
+   > ```
+   >
+   > Where the value in `--secret` is the secret value you place in `GITHUB_WEBHOOK_SECRET` described below, and `--repo` is the repo you want to test against.
```

Comment:
IIRC, the default token generated by `gh auth login` will have full permissions. `gh webhook` just needs the ability to alter webhook settings on the repo, which that token should have. At least in my testing, it "just worked" without me needing to do anything special (other than signing up for the beta).

I'm not sure it would really be feasible to set it up on CI. I think it would need to start by creating a bot account, and manually creating a token with a limited scope to a single "test" repo, and add that token to the workflow secrets. But then I think PR authors would then have full write access to that repo, which could be dangerous.

I think it would be challenging to do in a secure way, but I haven't thought about it much. I don't know when it will be out of beta, either.